src/pages/workspaces/workspace/WorkspaceMenu.test.js

```diff
@@ -263,7 +296,13 @@ describe('WorkspaceMenu - Azure workspace', () => {
     true, false
   ])('enables/disables Share menu item based on canShare: %s', canShare => {
     // Arrange
-    useWorkspaceDetails.mockReturnValue({ workspace: { canShare, workspace: {} } })
+    useWorkspaceDetails.mockReturnValue({
```

Comment:
Again, test bug-- the workspace details being provided mimic a GCP workspace instead of an Azure one.